.golangci.yml

```diff
@@ -0,0 +1,23 @@
+linters-settings:
+  goimports:
+    local-prefixes: github.com/golangci/golangci-lint
+  govet:
+    check-shadowing: false
+
+  nolintlint:
+    allow-leading-space: true # don't require machine-readable nolint directives (i.e. with no leading space)
+    allow-unused: false # report any unused nolint directives
+    require-explanation: false # don't require an explanation for nolint directives
+    require-specific: false # don't require nolint directives to be specific about which linter is being skipped
+
+linters:
+  disable-all: true
+  enable:
+    - gofmt
+    - gci
+    - goimports
+    - govet
+
+run:
+  timeout: 10m
```

Comment:
I think that 10min might be too much, I would leave the default 1min

Reply:
I reduce to 5 minute